pkg/operator/defaultstorageclass/controller.go

```diff
@@ -196,6 +196,10 @@ func newStorageClassForCluster(infrastructure *configv1.Infrastructure) (*storag
 		storageClassFile = "storageclasses/gcp.yaml"
 	case configv1.VSpherePlatformType:
 		storageClassFile = "storageclasses/vsphere.yaml"
+	case configv1.AzurePlatformType:
+		return nil, supportedByCSIError
```

Comment:
Hold on, should it create in-tree StorageClass for Azure File? Its CSI migration is not GA yet.

Reply:
Scratch this, we did not create StorageClass for Azure File